modules/custom/az_publication/az_publication.install

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for az_publication module.
+ */
+
+/**
+ * Install the new az_publication supporting module az_curated_views.
```

Comment:
This wording is awkward, Chat GPT, help!!!

Reply:
Maybe this?
```suggestion
 * Install the new az_curated_views module (now required by az_publication).
```